spyder/widgets/helperwidgets.py

```diff
@@ -375,6 +375,65 @@ def filterAcceptsRow(self, row_num, parent):
             return True
 
 
+class MultipleSortFilterProxy(QSortFilterProxyModel):
```

Comment:
Maybe this should be named `ShortcutsSortFilterProxy` and should go in `spyder/preferences/shortcuts.py`?

Reply:
I put it in here just in case someday we need a multiple sort filter in other place, should I move it?